apps/i18n/common/en_us.json

```diff
@@ -770,13 +770,15 @@
   "lessonNumber": "Lesson Number",
   "lessonNumbered": "Lesson {lessonNumber}: {lessonName}",
   "lessonPlans": "Lesson Plans",
+  "lessonsAttempted": "Lessons attempted in ",
```

Comment:
I could be wrong, but I think trailing white spaces might be stripped from this file.

Reply:
Oh good idea - I will do that

Reply:
Actually not sure if I can do that since the scriptFreindlyName has to be a link